src/EmailService.Data.Provider.MsSql.Ef/Migrations/20211011124100_InitialCreate.cs

```diff
@@ -91,6 +91,27 @@ protected override void Up(MigrationBuilder builder)
         {
           table.PrimaryKey($"PK_{DbUnsentEmail.TableName}", x => x.Id);
         });
+
```

Comment:
you can not add anything to DB when you add via the already applied migration. Create new migration

Reply:
fixed